src/DurableTask.Core/Common/Utils.cs

```diff
@@ -62,6 +62,90 @@ public static class Utils
             Binder = new PackageUpgradeSerializationBinder()
 #endif
         };
+        private static readonly JsonSerializer objectJsonSerializer = JsonSerializer.Create(ObjectJsonSettings);
+
+
+        private static readonly JsonSerializer serializer = JsonSerializer.Create();
```

Comment:
```suggestion
        private static readonly JsonSerializer DefaultSerializer = JsonSerializer.Create();
```
And please move this up with the other `static readonly` fields.

Reply:
I applied the renaming, but I was unsure about your suggestion on moving this further up, as it seems to me that all the fields preceding `DefaultSerializer` are already `static readonly`, and near the top of the class. Am I missing something?